src/components/MovieItem/index.js

```diff
@@ -9,10 +9,10 @@ const MovieItem = ({
         movie.poster_path ? `${process.env.REACT_APP_MOVIE_BASE_URL}/w342/${movie.poster_path}` : "";
     return (
         // we can have multiple clickable elements, each of them can have a handler provided by the parent
-        <div className={"card border-0"} onClick={() => posterOnClickHandler(movie)}>
-            <img className={"card-img-top"} src={posterPath} alt={"Poster Not Found"}/>
-            <div className={"card-body"}>
-                <h5 className={"card-title fs-6"}>{movie.title}</h5>
+        <div className={"card border-0 bg-light h-100"} onClick={() => posterOnClickHandler(movie)}>
```

Comment:
I don't these changes work. I checked the page, the image is stretched too much.

Reply:
Check it again